core/README.md

```diff
@@ -90,10 +90,7 @@ There may be a lot of variation on how these images are built. This is what
 worked:
 
 - Use Windows OS. Docker Desktop might also work on Mac, but not on Linux.
-- Download Docker Desktop _Edge_. At the time of writing (Nov 2019), only the
```

Comment:
I'm not using Docker Desktop, but this Nov 2019 description looks outdated. Let me know if the required features are still available in Edge distribution only under experimental flags.

Reply:
i don't think anything special is needed to run docker container. It is Docker Desktop for Mac & Windows and `docker engine` on Linux